### ISSUE TITLE
train-diqa-base.ipynb file - Runtime error at Step 3

### ISSUE THREAD

**Issue by suresh-s** [ocampor/image-quality#18]
**Describe the bug**
I try to execute train-diqa-base.ipynb file, Failure noticed

**To Reproduce**
1. cloned Image-quality project, and change directory to notebooks
2. Launch the Jupyter notebook server
3. Selected train-diqa-base.ipynb file to run
4. Step 1 and Step 2 are executed
5. Step 3  has following code snippet
```
builder = imquality.datasets.LiveIQA()
builder.download_and_prepare()
```

I am getting following issue

```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-3-1dbb51f702d0> in <module>
----> 1 builder = imquality.datasets.LiveIQA()
      2 builder.download_and_prepare()

/anaconda3/lib/python3.6/site-packages/tensorflow_datasets/core/api_utils.py in disallow_positional_args_dec(fn, instance, args, kwargs)
     50     _check_no_positional(fn, args, ismethod, allowed=allowed)
     51     _check_required(fn, kwargs)
---> 52     return fn(*args, **kwargs)
     53 
     54   return disallow_positional_args_dec(wrapped)  # pylint: disable=no-value-for-parameter

/anaconda3/lib/python3.6/site-packages/tensorflow_datasets/core/dataset_builder.py in __init__(self, data_dir, config, version)
    195     else:  # Use the code version (do not restore data)
    196       logging.info("Load pre-computed datasetinfo (eg: splits) from bucket.")
--> 197       self.info.initialize_from_bucket()
    198 
    199   @utils.memoized_property

/anaconda3/lib/python3.6/site-packages/tensorflow_datasets/core/utils/py_utils.py in __get__(self, obj, objtype)
    131     cached = getattr(obj, attr, None)
    132     if cached is None:
--> 133       cached = self.fget(obj)
    134       setattr(obj, attr, cached)
    135     return cached

/anaconda3/lib/python3.6/site-packages/tensorflow_datasets/core/dataset_builder.py in info(self)
    252           "Otherwise, the created .info may not match the info version from "
    253           "the restored dataset.")
--> 254     return self._info()
    255 
    256   @api_utils.disallow_positional_args

~/Documents/source/image-quality/imquality/datasets/live_iqa.py in _info(self)
     74             citation=CITATION,
     75             redistribution_info={
---> 76                 'license': LICENSE
     77             }
     78         )

/anaconda3/lib/python3.6/site-packages/tensorflow_datasets/core/api_utils.py in disallow_positional_args_dec(fn, instance, args, kwargs)
     50     _check_no_positional(fn, args, ismethod, allowed=allowed)
     51     _check_required(fn, kwargs)
---> 52     return fn(*args, **kwargs)
     53 
     54   return disallow_positional_args_dec(wrapped)  # pylint: disable=no-value-for-parameter

/anaconda3/lib/python3.6/site-packages/tensorflow_datasets/core/dataset_info.py in __init__(self, builder, description, features, supervised_keys, homepage, urls, citation, metadata, redistribution_info)
    141     if urls:  # TODO(epot):Delete field once every user have been migrated
    142       raise ValueError("`urls=` field is deprecated. Please use "
--> 143                        "`homepage='{}'` instead.".format(urls[0]))
    144     if homepage:
    145       self._info_proto.location.urls[:] = [homepage]

ValueError: `urls=` field is deprecated. Please use `homepage='https://live.ece.utexas.edu/research/quality/subjective.htm'` instead.
```

**Comment by titsitits** [comment]
Hi,

I got the same issue, while trying your notebook in Google colab (with tensorflow version 2.1.0-rc1 and python3.6).

Maybe some indications on the specific environment used for this notebook would be interesting.

I made a workaround by replacing the defective line:

**In file /content/image-quality/imquality/datasets/live_iqa.py,
change line 73 (urls=URLS) with:
homepage=URLS[0]**

The following of the notebook seems to work correctly then. If you want to change this line programmatically in the notebook, something like this should work (juste change the path accordingly):

!sed -i 's/urls=URLS,/homepage=URLS[0],/g' "/content/image-quality/imquality/datasets/live_iqa.py"

Mickael
